### PR TITLE
Add Java device type API

### PR DESCRIPTION
**Pull request by bluemarvin**

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java [hunk 1]
```diff
@@ -734,6 +735,13 @@ public int getPointerColor() {
         return SettingsStore.getInstance(this).getPointerColor();
     }
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: can we add a method for [`getDeviceType()`](https://github.com/MozillaReality/FirefoxReality/pull/1097/files#diff-94535c76764a3a4f9a3d7cbf496c0d7dR21) than returns a string for each (e.g., `"unknown"`, `"oculus-go"`, etc.)? or should I just import `DeviceType` directly?

File: app/src/common/shared/org/mozilla/vrbrowser/input/DeviceType.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package org.mozilla.vrbrowser.input;
+
+import android.util.Log;
+
+public class DeviceType {
+    // These values need to match those in Device.h
+    public static final int Unknown = 0;
+    public static final int OculusGo = 1;
+    public static final int OculusQuest = 2;
```
Comment (cvan) [inline app/src/common/shared/org/mozilla/vrbrowser/input/DeviceType.java]: we should probably remember to add the others, but this is fine for now

Comment (bluemarvin) [reply app/src/common/shared/org/mozilla/vrbrowser/input/DeviceType.java]: I only wanted to add what we needed.

File: app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java (hunk 1 above)
Comment (bluemarvin) [reply app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: You should just be using `DeviceType.getDeviceType()` directly.

**Pull request merged**